docs/contents.md

```diff
@@ -67,21 +65,27 @@ The **Outbreak Response Framework** solution pack contains the following resourc
+| Find and Link Affected Assets                               | Examine vulnerabilities associated with the outbreak CVEs, and if any are found, identify the affected assets and link them to the outbreak.    |
+| Remediate and Mitigate Outbreak Response Alert              | Block all types of outbreak response alert indicators on the firewall according to their block status and take steps to mitigate them.          |
+| Outbreak Alert Time Frame Analysis                          | Identify and deactivate outbreak alerts whose investigation time frame has expired.                                                             |
+| Automated Deployment: Outbreak Alert Response Solution Pack | Automate Installation of Recent Outbreak Alert Solutions Pack (Last x Days) with Email Notification                                             |
+| > Automated Deployment > Get Outbreak CVEs and IOC Details  | Retrieve Outbreak CVEs and IOC details, and set the last investigation time for newly installed outbreak alerts.                                |
+| Configure Outbreak Response Framework                       | Set up the outbreak response framework, configure global value key store entries, and set up schedules for use in outbreak investigations.      |
+| Get Outbreak CVEs and IOCs Details                          | Obtain outbreak alert CVE details from NIST and IOC information from Fortinet FortiGuard.                                                       |
+| Find Known Exploited Vulnerabilities (KEV) CVEs             | Fetch Known Exploited Vulnerabilities (KEV) CVEs from NIST and create CVE record.                                                               |
+| Create IOCs as Threat Intel Feeds                           | Obtain the IOCs from Fortinet FortiGuard and create the threat feeds in the Threat Intel module for the specified outbreak alert.               |
+| Update Outbreak Alert Details![new](./res/icon-new.svg)     | Fetches and Updates the Outbreak Alert's Details                                                                                                |
+| Update CVE Details from NIST![new](./res/icon-new.svg)      | Fetches CVE Details from NIST and Updates the CVE record                                                                                        |
+| Tracking Outbreak: Retrieve CVEs and IOCs                   | Fetch outbreak alert CVEs (KVE) from NIST and IOCs from Fortinet FortiGuard, specifically focusing on those marked with a status of 'Tracking'. |
+| Reset Outbreak to Tracking                                  | Set the deactivated and resolved outbreak alert to tracking                                                                                     |
+| Bulk Ingest Outbreak Alert IOCs                             | Bulk Ingest Outbreak Alert IOCs on the basis of Outbreak Tag                                                                                    |
```

Comment:
Two new Playbooks are added with this
1. Link Related Threat Intel Feeds - This playbook links the unlinked Threat Intel Feeds to the Outbreak Alert
2. Find Threat Intel Feeds Records Linked to Outbreak Alert - Finds every Threat Intel Feeds records linked to Outbreak Alert